### DISCUSSION TITLE
It will be helpful for some people to add a arg for remove emoji.

### DISCUSSION THREAD

**Discussion by techoi** [vercel/vercel#8775] [Ideas]
https://github.com/vercel/vercel/issues/8644

If there is --remove-emoji arg to replace emoji to label of emoji, it will be helpful for some people.

For example type `vercel login --remove-emoji` then the OUTPUT should 

> Congratulations! You are now logged in. In order to deploy something, run `vercel`.
> [tip]  Connect your Git Repositories to deploy every branch push automatically (https://vercel.link/git).

 instead of

> Congratulations! You are now logged in. In order to deploy something, run `vercel`.
> 💡  Connect your Git Repositories to deploy every branch push automatically (https://vercel.link/git).

I want to contribute this feature for someone else.

Maybe the way is..
- Add arg to `arg-commmon.ts`
- Add property removeEmoji to createOutput in `/cli/src/index.ts`
- Change `print` method in create-output.ts to replace emoji to label.

Can I try it?

If there is better way please let me know.
Thanks

**Comment by amyegan** [comment]
Hello @techoi! We ❤️ community contributions! Before you start building anything though, can you tell me more about why you want to have this feature? For example, if this is an accessibility problem or workflow blocker for you then it's probably an issue for others as well. And we want to know about it so we can work with you to make the CLI better for everyone 🙏

**Reply by techoi** [reply]
Hello @amyegan! Thank you for answering me.

I wan to working on this feature for someone like Pound-Hash who wrote this issue(https://github.com/vercel/vercel/issues/8644).
> Yellow on white is hard on the eyes for many. Green on white might be even worse for colorblind users (like myself). Can we have the option to remove the emojis from the output, please?

Actually I am not the person who has this problem. However i was motivated to contribute vercel.

**Reply by amyegan** [reply]
Thanks for sharing that issue link! I checked with the CLI team, and they would be happy to have you contribute this feature. They suggested that it might be more appropriately named `--text-only`. I think they're leaning toward simply removing the emoji rather than replacing it with a text description. But if you feel strongly about replacement instead of removal, I'm sure they would be willing to listen. Just be sure to link back to this discussion and the original issue in the PR description. And be prepared for questions in the PR review! 🙇

**Comment by techoi** [comment]
https://github.com/vercel/vercel/pull/8826
